src/components/SideBar.jsx

```diff
@@ -0,0 +1,223 @@
+import styled, { css } from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { NavLink } from 'react-router-dom'
+import { useRef } from 'react'
+
+import {
+  getBreakpointsStyles,
+  getDesktopStyles,
+  useBreakpointValue,
+  ScreenWrapper,
+} from '../styles/breakpoints'
+import { CloseIcon, UserCheckIcon, CalendarCheckOutIcon, LogOutIcon } from '../assets/icons'
+import { AppLogo } from './AppLogo'
+import { PrimaryColorBtn } from './PrimaryColorBtn'
+
+export const SideBar = ({
+  isBurgerMenuOpen,
+  setIsBurgerMenuOpen,
+  theme,
+  selectedTab,
+  setSelectedTab,
+}) => {
+  const owerlayRef = useRef()
+  const { t } = useTranslation()
+  const iconCheckSize = useBreakpointValue({ mobileValue: 20, tabletValue: 24, desktopValue: 24 })
+  const iconLogOutSize = useBreakpointValue({ mobileValue: 18, tabletValue: 20, desktopValue: 20 })
+  const iconClozeSize = useBreakpointValue({ mobileValue: 25, tabletValue: 33 })
+
+  const IconCheckColor = (isSelected) => {
+    if (isSelected) {
+      return theme === 'light' ? '#3e85f3' : '#fff'
+    }
+    return theme === 'light' ? ' rgba(52, 52, 52, 0.5)' : '#fff'
+  }
+
+  const handleCloseBurgerMenu = () => {
+    setIsBurgerMenuOpen(false)
+  }
+
+  const handleOwerlayClick = (e) => {
+    if (e.target === owerlayRef.current) {
+      handleCloseBurgerMenu()
+    }
+  }
+
+  const handleChangeTab = (type) => {
+    setSelectedTab(type)
```

Comment:
No need to create additional functions if you will need only one like. Please remove handleCloseBurgerMenu and handleChangeTab. If the future we can add if it will be needed